packages/nx/src/command-line/init/init-v2.ts

```diff
@@ -74,16 +74,27 @@ export async function initHandler(options: InitArgs): Promise<void> {
 
   const { plugins, updatePackageScripts } = await detectPlugins();
 
+  const packageJson: PackageJson = readJsonFile('package.json');
   if (!plugins.length) {
     // If no plugins are detected/chosen, guide users to setup
     // their targetDefaults correctly so their package scripts will work.
-    const packageJson: PackageJson = readJsonFile('package.json');
     if (isMonorepo(packageJson)) {
       await addNxToMonorepo({ interactive: options.interactive });
     } else {
       await addNxToNpmRepo({ interactive: options.interactive });
     }
   } else {
+    if (isMonorepo(packageJson)) {
+      await addNxToMonorepo({
+        interactive: options.interactive,
+        nxCloud: false,
+      });
+    } else {
+      await addNxToNpmRepo({
+        interactive: options.interactive,
+        nxCloud: false,
+      });
+    }
```

Comment:
Why are we explicitly setting cloud to false here? On second thought, I think we can get rid of the plugins.length check entirely since these call the same functions now?

Reply:
We pass `false` to skip the prompt which is repeated down below.